Jenkinsfile

```diff
@@ -75,6 +75,27 @@ pipeline {
       }
 
       stages {
+        stage('simulator') {
+          agent { docker { image 'ghcr.io/commaai/openpilot-base'; args '--user=root' } }
```

Comment:
any reason this should use openpilot-base? we should use something that already has docker. perhaps we can use GitHub Action's ubuntu runner image if they publish it?